docs/operations/libraries/workflow_alpha_release.md

```diff
@@ -0,0 +1,21 @@
+## Alpha Release Workflow
+This describes the process for pushing an alpha release for a `library` module.
+A `library` module is a module or repository that is a dependency of one or more
+Neon `services` or `plugins`.
+
+### Application
+This process applies when making any change to a `library` module.
+
+### Process
+- Create a branch from the `dev` branch of the module.
+- Complete any changes, including test coverage, documentation updates, and
+  dependency changes.
+- Create a Pull Request to the `dev` branch.
+- If any tests fail, mark the Pull Request as a Draft and make necessary changes.
+- After tests are passing, request a review from a maintainer.
+  > If the PR was marked as a draft, mark it ready for review here
+- Upon approving review, `Squash and Merge` changes to the `dev` branch
```

Comment:
Worth noting that in the squash, attention should be paid to the commit messages? Not sure what your intention is with any kind of automated changelogs.

Reply:
Ideally, the PR template is enough and I'm not using any kind of conventional commits (yet). This is probably something to note in the review process though to validate a PR title/description